app/controllers/dashboard_controller.rb

```diff
@@ -8,9 +8,9 @@ def index
 		@my_patrol = selected_user.patrol
 
 		@activities = PublicActivity::Activity.order("created_at desc").limit(10)
-
-		@confirmed_offers = Offer.joins(:roster).where('status = ? AND rosters.organisation = ?', 'accepted', selected_user.organisation).order(updated_at: :desc).limit(3)
-
+		confirmed_offer = Offer.where(roster_id: nil).where('status = ? ', 'accepted').order(updated_at: :desc).limit(2)
+		confirmed_offer1 = Offer.joins(:roster).where('status = ? AND rosters.organisation = ?', 'accepted', selected_user.organisation).order(updated_at: :desc).limit(3)
```

Comment:
More descriptive variable name please.  
confirmed_offer might be confirmed_offer_1way
confirmed_offer1 might be confirmed_offer_reciprocal